src/user/screens/profile.screen.js

```diff
@@ -84,10 +80,11 @@ class Profile extends Component {
       isPendingUser,
       isPendingOrgs,
       isPendingCheckFollowing,
-      navigation
+      navigation,
```

Comment:
That's right, I was thinking about adding CircleCI or Travis in here, at least checking for eslint would be nice!